nomad/state/paginator/tokenizer.go

```diff
@@ -33,27 +33,28 @@ type CreateIndexGetter interface {
 // StructsTokenizerOptions is the configuration provided to a StructsTokenizer.
 //
 // These are some of the common use cases:
+//
 //   - Structs that can be uniquely identified with only its own ID:
 //
-//       StructsTokenizerOptions {
-//           WithID: true,
-//       }
+//     StructsTokenizerOptions {
+//     WithID: true,
+//     }
 //
 //   - Structs that are only unique within their namespace:
 //
-//       StructsTokenizerOptions {
-//           WithID:        true,
-//           WithNamespace: true,
-//       }
+//     StructsTokenizerOptions {
+//     WithID:        true,
+//     WithNamespace: true,
+//     }
 //
 //   - Structs that can be sorted by their create index should also set
 //     `WithCreateIndex` to `true` along with the other options:
 //
-//       StructsTokenizerOptions {
-//           WithID:          true,
-//           WithNamespace:   true,
-//           WithCreateIndex: true,
-//       }
+//     StructsTokenizerOptions {
+//     WithID:          true,
+//     WithNamespace:   true,
+//     WithCreateIndex: true,
+//     }
```

Comment:
Weird 😬

Reply:
figured it out - need to indent by 4 spaces to be in a code block